server/region_handler.go

```diff
@@ -43,11 +43,12 @@ import (
 
 const (
 	pDBName    = "db"
-	pTableName = "table"
+	pHexKey    = "hexKey"
 	pRegionID  = "regionID"
 	pRecordID  = "recordID"
 	pStartTS   = "startTS"
-	pHexKey    = "hexKey"
+	pTableID   = "table_id"
```

Comment:
Should we need to use the unified format? `tableID` or `region_id`

Reply:
It's based on @AndreMouche 's document. I'll check it out.

Reply:
Has moved it to a standalone part.